pages/studentPortal/index.tsx

```diff
@@ -65,6 +72,37 @@ export default function StudentPortalPage() {
     });
   }, [user]);
 
+  const [updateLastSeenModal] = useMutation(UPSERT_LAST_SEEN_MODAL);
+  const [showModal, setShowModal] = useState(false);
+
+  useQuery<FetchModalData>(FETCH_LAST_SEEN_MODAL, {
+    variables: {
+      _id: user.uid,
```

Comment:
use the skip option along with the current users role to conditionally execute this query: https://stackoverflow.com/questions/59489097/use-query-hook-react-apollo-conditional-call

Reply:
great idea.  it works.